database/migrations/2020_10_23_161736_fix_zero_values_for_locations.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+use App\Models\User;
+use App\Models\Asset;
+
+class FixZeroValuesForLocations extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        App\Models\Asset::where('location_id', '=', '0')
+            ->update(['location_id' => null]);
+
+        App\Models\Asset::where('rtd_location_id', '=', '0')
+            ->update(['rtd_location_id' => null]);
+
+        App\Models\User::where('location_id', '0')
```

Comment:
Just to be consistent with the above two, I'd very slightly prefer to `'='` parameter here, too. But not so much so that I would hitch up this PR.

Reply:
Ah, thanks - that was just an oversight